example/index.html

```diff
@@ -125,11 +126,10 @@
           },
 
           // function retrieving an array of objects
-          values: function (text, cb) {
-            remoteSearch(text, function (users) {
-              cb(users);
-            });
-          },
```

Comment:
@mrsweaters, why did you decide to remove the removeSearch functionality in the example? If mockbin stopped working, you could just show fake data.

As it stands now, the heading `Tribute with a local collection (on <code>@</code>) and a remote one (on <code>#</code>):` isn't correct any more. And the possibility of providing values through a function isn't documented. The latter is something I probably forgot about in my original PR.